### DISCUSSION TITLE
Documentation

### DISCUSSION THREAD

**Discussion by jgrss** [jgrss/geowombat#202] [Ideas]
@mmann1123 Following on one of you PR comments, we need to update the online documentation. Let's make an inventory of methods that are not covered, partially covered, or not clearly defined.

**Comment by jgrss** [comment]
- The differences between `save()` and `to_raster()` is not clearly illustrated.
- Comb the API to check coverage of methods
- Give better and updated examples of existing methods
- Add examples of methods not covered

**Comment by mmann1123** [comment]
Do we do this in Discussions or in a Projects checklist? 
- [ ] `gw.coregister` isn't covered at all I think. 
- [ ] `netcdf`

**Reply by jgrss** [reply]
You're right, let's move this into Projects.

**Reply by jgrss** [reply]
[Project page](https://github.com/users/jgrss/projects/1/views/1)
